admin/partials/wp-security-bp-admin-display.php

```diff
@@ -14,12 +14,115 @@
 ?>
 
 <!-- This file should primarily consist of HTML with a little bit of PHP. -->
-<h1>OLAKEASE</h1>
 <?php
 
-$users = get_users( array( 'include' => array( 2, 3 ) ) );
-foreach ( $users as $user ) {
-    if ( true ) {
-        var_dump( $user );
+class WP_Security_BP_Admin_Get_Users {
+
+	private $users;
+
+	public function __construct(){
+		$this->$users = get_users(); 
+	}
+	
+	public function check_danger_id_admin(){ 
+
+		$danger_ids = array();
+
+			foreach( $this->$users as $user ){
+				if( $user->ID < 5 ){
+					$result = [
+						'user_id'    => $user->ID,
+						'user_name'  => $user->display_name,
+						'user_email' => $user->user_email,
+					];
+					array_push($danger_ids, $result);
+				}
+			}
+
+		return $danger_ids;
+	}
+}
+
+?>	
+<!--------- V I E W --------->
+<h2>WP SECURITY PLUGIN</h2>
+<?php 
+
+$check = new WP_Security_BP_Admin_Get_Users();
+$check_danger_id_admin = $check->check_danger_id_admin();
```

Comment:
I would include the instantiation of the Users class inside the Admin class. This is more oriented to have the font-end built with a JavasCript framework instead of PHP + HTML.